shell/server/drivers/ip.js

```diff
@@ -121,9 +122,14 @@ IpInterfaceImpl = class IpInterfaceImpl {
   }
 };
 
-makeIpInterface = (persistentMethods) => {
-  return new Capnp.Capability(new IpInterfaceImpl(persistentMethods), IpRpc.PersistentIpInterface);
-};
+// TODO(cleanup): Meteor.startup() needed because 00-startup.js runs *after* code in subdirectories
+//   (ugh).
```

Comment:
Should server/00-startup.js be moved to server/lib/00-startup.js?

https://guide.meteor.com/structure.html#load-order